### PR TITLE
examples: add upgrade YAML for MicroOS

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by aplanas** [comment]
This PR is ready for review

**Comment by aplanas** [comment]
I tested this, and I found one issue with selinux (bsc#1210822) and k3s-selinux (bsc#1210824). Only the first one is important, but this plan has been tested and drive the installation until the end.

**Comment by aplanas** [comment]
cc @brandond

**Review by brandond — APPROVED** [review]

File: examples/openSUSE/microos.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: microos
+  namespace: system-upgrade
+spec:
+  concurrency: 2
+  nodeSelector:
+    matchExpressions:
+      - {key: node-role.kubernetes.io/worker, operator: In, values: ["true"]}
```
Comment (brandond) [inline examples/openSUSE/microos.yaml:29]: Are you sure about this? RKE2 and K3s do not use the 'worker' role so this will never match any nodes.

Comment (aplanas) [reply examples/openSUSE/microos.yaml:29]: I label it as such, but I used the same that is in https://github.com/rancher/system-upgrade-controller/blob/master/examples/suse/sles.yaml#L26.

Can something like this be better? https://github.com/rancher/system-upgrade-controller/blob/master/examples/ubuntu/bionic.yaml#L31

Comment (brandond) [reply examples/openSUSE/microos.yaml:29]: I don't know, none of these labels will exist by default - I guess the examples assume people understand that they need to label their nodes to match the LabelSelector in the plan? I don't mean to make you fix all of this; I approved so you can merge if you want .

Comment (aplanas) [reply examples/openSUSE/microos.yaml:29]: Excellent. I do not have rights to merge it. Can you do that?

Thanks!

**Pull request merged**